shared/teams/team/rows/invite-row/request/index.tsx

```diff
@@ -59,21 +55,88 @@ export const TeamRequestRow = (props: Props) => {
           open={props.isRolePickerOpen}
           disabledRoles={props.disabledReasonsForRolePicker}
         >
-          <Button label="Let in as..." onClick={onAccept} small={true} style={styles.letInButton} />
+          <Kb.Button label="Let in as..." onClick={onAccept} small={true} style={styles.letInButton} />
         </FloatingRolePicker>
-        <Button
+        <Kb.Button
           label="Ignore"
           onClick={onIgnoreRequest}
           small={true}
           style={styles.ignoreButton}
           type="Danger"
         />
-        {!Styles.isMobile && <Icon onClick={onChat} style={styles.icon} type="iconfont-chat" />}
-      </Box>
-    </Box>
+        {!Styles.isMobile && <Kb.Icon onClick={onChat} style={styles.icon} type="iconfont-chat" />}
+      </Kb.Box>
+    </Kb.Box>
   )
 }
 
+const TeamRequestRowNew = (props: Props) => {
+  const {ctime, fullName, username, onAccept, onOpenProfile} = props
+
+  const {showingPopup, setShowingPopup, toggleShowingPopup, popup, popupAnchor} = Kb.usePopup(attachTo => (
+    <Kb.FloatingMenu
+      items={[
+        {onClick: props.onChat, title: 'Chat'},
+        {danger: true, onClick: props.onIgnoreRequest, title: 'Deny'},
+      ]}
+      visible={showingPopup}
+      onHidden={() => setShowingPopup(false)}
+      closeOnSelect={true}
+      attachTo={attachTo}
+      position="bottom left"
+      positionFallbacks={['top left']}
+    />
+  ))
+
+  return (
+    <Kb.Box style={styles.container}>
+      <Kb.ClickableBox style={styles.clickContainer} onClick={() => onOpenProfile(username)}>
+        <Kb.Avatar username={username} size={Styles.isMobile ? 48 : 32} />
+        <Kb.Box style={styles.userDetails}>
+          <Kb.ConnectedUsernames type="BodySemibold" colorFollowing={true} usernames={[username]} />
+          <Kb.Box style={Styles.globalStyles.flexBoxRow}>
+            <Kb.Meta title="please decide" style={styleCharm} backgroundColor={Styles.globalColors.orange} />
+            {!Styles.isMobile && (
+              <Kb.Text type="BodySmall" lineClamp={1}>
+                {fullName !== '' && `${fullName}  • `}
+                {moment(ctime * 1000).fromNow()}
+              </Kb.Text>
+            )}
+          </Kb.Box>
+        </Kb.Box>
+      </Kb.ClickableBox>
+      <Kb.Box style={styles.floatingRolePickerContainer}>
```

Comment:
I think `styles.container` is making it so these buttons still go below on mobile. 
<img width="319" alt="image" src="https://user-images.githubusercontent.com/11968340/74167251-f6541e00-4bf5-11ea-97cf-8db3ec7f7152.png">